mpa/deploy/utils/mmdet_symbolic.py

```diff
@@ -0,0 +1,432 @@
+# Copyright (C) 2022-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
```

Comment:
Is this whole file needed for naive IR export?
Copied from somewhere or your own implementation?

Reply:
I copied it from forked mmdet.